### ISSUE TITLE
Define function in include

### ISSUE THREAD

**Issue by gsemet** [casey/just#2512]
Hello

I often see myself fetching some remote bash script just to have access to a few function in my recipes. Would it be possible to define a few bash functions directly in justfiles ?

Thanks

**Comment by laniakea64** [comment]
Yes it is possible, but specifically how the best way to do it can vary depending on usage.  Could you please provide an example justfile where you're currently using remote bash script?

See https://github.com/casey/just/discussions/1707 for some possible ideas

**Comment by casey** [comment]
Also see #2079, which would allow defining a shell prelude to be executed before all shell invocations.
